src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/adapter/AuthService.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.authorizationserver.adapter
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import reactor.core.publisher.Mono
+
+@Suppress("SpringJavaInjectionPointsAutowiringInspection")
+@Service
+class AuthService(
+  @Qualifier("authWebClient") private val webClient: WebClient,
+  @Value("\${auth.enabled:false}") private val authEnabled: Boolean,
```

Comment:
What is this used for? Not sure we need a feature shield for Auth?

Reply:
Removed 'authEnabled'